pkg/operator/ceph/cluster/mon/health_test.go

```diff
@@ -58,7 +58,7 @@ func TestCheckHealth(t *testing.T) {
 	c.waitForStart = false
 	defer os.RemoveAll(c.context.ConfigDir)
 
-	c.mapping.Node["mon1"] = &NodeInfo{
+	c.mapping.Node["rook-ceph-mon1"] = &NodeInfo{
```

Comment:
did the test require the `rook-ceph` prefix or just changing for consistency with production?

Reply:
It only adds consistency in general for the tests and production, because the tested operator code started mons are having `rook-ceph-mon%d` names and the mocked ones were most of the time `mon%d`.
I can move it out to another PR if wanted. :)

Reply:
as a separate commit in the same PR is fine with me, just curious if it was for consistency